diff --git a/node_modules/pdf-poppler/index.js b/node_modules/pdf-poppler/index.js
index ac24035..47b2cd2 100644
--- a/node_modules/pdf-poppler/index.js
+++ b/node_modules/pdf-poppler/index.js
@@ -3,10 +3,10 @@ const path = require('path');
 const {spawn} = require('child_process');
 
 let platform = os.platform();
-if (!['darwin', 'win32'].includes(platform)) {
-    console.error(`${platform} is NOT supported.`);
-    process.exit(1);
-}
+//if (!['darwin', 'win32'].includes(platform)) {
+//    console.error(`${platform} is NOT supported.`);
+//    process.exit(1);
+//}
 
 let popplerPath;
 
@@ -64,10 +64,10 @@ else if (platform === 'darwin') {
     spawn('install_name_tool', ['-change', `/usr/local/Cellar/poppler/0.66.0/lib/libpoppler.77.dylib`, `${path.join(dyldPath, 'libpoppler.77.0.0.dylib')}`, `${path.join(popplerPath, 'pdftocairo')}`]);
     spawn('install_name_tool', ['-change', `/usr/local/Cellar/poppler/0.66.0/lib/libpoppler.77.dylib`, `${path.join(dyldPath, 'libpoppler.77.0.0.dylib')}`, `${path.join(popplerPath, 'pdfimages')}`]);
 }
-else {
-    console.error(`${platform} is NOT supported.`);
-    process.exit(1);
-}
+//else {
+//    console.error(`${platform} is NOT supported.`);
+//    process.exit(1);
+//}
 
 module.exports.path = popplerPath;
 module.exports.exec_options = execOptions;
